R/guide-colorbar.r

```diff
@@ -244,7 +244,7 @@ guide_geom.colorbar <- function(guide, layers, default_mapping) {
   guide_layers <- lapply(layers, function(layer) {
     matched <- matched_aes(layer, guide, default_mapping)
 
-    if (length(matched) && ((is.na(layer$show.legend) || layer$show.legend))) {
+    if (length(matched) && (isTRUE(is.na(layer$show.legend)) || isTRUE(layer$show.legend))) {
```

Comment:
Should `length(matched)` get an explicit comparison, like `length(matched) > 0` (I assume that's what is meant)?

Reply:
Agreed, thanks.